common/types.go

```diff
@@ -310,3 +318,9 @@ func GetIdLength(id string) int64 {
 	existingBlockId, _ := base64.StdEncoding.DecodeString(id)
 	return int64(len(existingBlockId))
 }
+
+func init() {
+	DefaultWorkDir = JoinUnixFilepath(GetDefaultWorkDir(), ".cloudfuse")
+	DefaultLogFilePath = JoinUnixFilepath(DefaultWorkDir, "cloudfuse.log")
+	StatsConfigFilePath = JoinUnixFilepath(DefaultWorkDir, "stats_monitor.cfg")
```

Comment:
Yes, these should. However, when I did that it broke other parts of the code and as I kept delving deeper I released there was a lot more lines that would need to be changed for this and it would make sense in a separate PR. In theory this doesn't matter as Go actually handles either delimiter on Windows. But yeah maybe a future backlog item to tidy these up.